Gui/opensim/Scripts/runTutorialTwo.py

```diff
@@ -47,14 +47,11 @@
 # Change the name of the model
 ##myModel.setName("Wrist Tendon Surgery.")
 
-
 ## Change the path points of the ECU_pre-surgery to match the existing ECU_post-surgery muscle
-for i in range(myModel.getMuscles().get("ECU_pre-surgery").getGeometryPath().getPathPointSet().getSize()):
-	# Get the Path Points from each muscle
-	ECUPrepathPoint			=	myModel.getMuscles().get("ECU_pre-surgery").getGeometryPath().getPathPointSet().get(i)
-	ECUPostpathPoint		=	myModel.getMuscles().get("ECU_post-surgery").getGeometryPath().getPathPointSet().get(i)
-	# Replace path point from ECU_pre-surgery to ECU_post-surgery
-	myModel.getMuscles().get("ECU_pre-surgery").getGeometryPath().replacePathPoint(myState,ECUPrepathPoint,ECUPostpathPoint)
+myModel.getMuscles().get("ECU_pre-surgery").getGeometryPath().updPathPointSet().clearAndDestroy();
+pps = myModel.getMuscles().get("ECU_pre-surgery").getGeometryPath().updPathPointSet();
+for i in range(myModel.getMuscles().get("ECU_post-surgery").getGeometryPath().getPathPointSet().getSize()):
+    pps.cloneAndAppend(myModel.getMuscles().get("ECU_post-surgery").getGeometryPath().getPathPointSet().get(i));
```

Comment:
It might be nice to use a local variable for the ECU_post-surgery PathPointSet so that these lines are shorter.

Reply:
Thanks, Chris. I incorporated your suggestions

Reply:
Looks good.